src/anomalib/utils/metrics/aupro.py

```diff
@@ -48,7 +48,7 @@ def __init__(
 
         self.add_state("preds", default=[], dist_reduce_fx="cat")  # pylint: disable=not-callable
         self.add_state("target", default=[], dist_reduce_fx="cat")  # pylint: disable=not-callable
-        self.register_buffer("fpr_limit", torch.tensor(fpr_limit))
+        self.register_buffer("fpr_limit", torch.tensor(fpr_limit), persistent=False)
```

Comment:
Please help double check whether removing it from the `state_dict` is reasonable, especially when we want to use it during testing.

Reply:
Thanks! I just pushed new commit to keep loading `fpr_limit` from `state_dict` by creating `AUPRO` instance before loading model, please take a look.